src/modules/Elsa.Workflows.Core/Middleware/Workflows/EngineExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,61 @@
+using Elsa.Common.Contracts;
+using Elsa.Workflows.Contracts;
+using Elsa.Workflows.Models;
+using Elsa.Workflows.Pipelines.WorkflowExecution;
+using Elsa.Workflows.State;
+using Microsoft.Extensions.Logging;
+
+namespace Elsa.Workflows.Middleware.Workflows;
+
+/// <summary>
+/// Adds extension methods to <see cref="ExceptionHandlingMiddleware"/>.
+/// </summary>
```

Comment:
Not a big deal (at all), but it gave me a giggle to see the effort that was made in the `WorkflowExecutionContext` where all the `///<summary>` lines were removed, but are added here 😄